test/use_text_editing_controller_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/src/framework.dart';
+import 'package:flutter_hooks/src/hooks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'mock.dart';
+
+void main() {
```

Comment:
Both tests should try to rebuild `HookBuilder` and check that the controller is the same and that the value has not changed.